src/plugins/discover/public/application/components/histogram/get_dimensions.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import moment from 'moment';
+import dateMath from '@elastic/datemath';
+import { IAggConfigs, TimeRangeBounds } from '../../../../../data/common';
+import { DataPublicPluginStart } from '../../../../../data/public';
+
+export function getDimensions(aggs: IAggConfigs, data: DataPublicPluginStart) {
+  const [metric, agg] = aggs.aggs;
+  const { from, to } = data.query.timefilter.timefilter.getTime();
+  agg.params.timeRange = {
+    from: dateMath.parse(from),
+    to: dateMath.parse(to, { roundUp: true }),
+  };
+  const bounds = agg.params.timeRange
+    ? data.query.timefilter.timefilter.calculateBounds(agg.params.timeRange)
+    : null;
+  // @ts-expect-error
```

Comment:
Dear valued reviewer, which type to use to omit  `//@ts-expect-error`,  couldn't  fine one (but also didn't spend hours, I must admit)